src/main.cpp

```diff
@@ -2542,9 +2547,9 @@ bool CheckBlock(const CBlock& block, CValidationState& state, bool fCheckPOW, bo
 
 bool ContextualCheckBlockHeader(const CBlockHeader& block, CValidationState& state, CBlockIndex * const pindexPrev)
 {
-    const Consensus::Params& consensusParams = Params().GetConsensus();
```

Comment:
@jtimon Of course, the loose reference to consensusParams is now causing an unused variable warning. It is not clear to me when you expect this to be fixed.